### PR TITLE
Blog Post: Theory Of Operation

### PR DESCRIPTION
**Pull request by squeedee**
A blog post explaining Cartographer behavior. Based on feedback and commonly asked questions

### CODE REVIEW

File: site/content/posts/theory-of-operation.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+excerpt: "Theory Of Operation"
+tags: ['faq', 'theory']
+---
+
+# Assumptions
+
+To explain Cartographer's operation, we're going to make some assumptions:
+
+1. You are familiar with creating and modifying Kubernetes resources
+2. You know the difference between a job and a pod.
```
Comment (davidmirror-ops) [inline site/content/posts/theory-of-operation.md:17]: In this context, would the difference be Job = short running process vs Pod = long running process?

**Comment by davidmirror-ops** [comment]
Thank you Rash, I believe this will be really useful. Especially the definitions of each one of the abstractions Cartographer use and how they look in an example Supply Chain

File: site/content/posts/theory-of-operation.md (hunk 1 above)
Comment (squeedee) [reply site/content/posts/theory-of-operation.md:17]: yes. I will get to terminating (short lived) processes later when I introduce runnable

**Comment by squeedee** [comment]
Going to reopen once we migrate to cartographer-site

**Pull request closed without merging**